.github/workflows/test.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        go-version: [1.20.x]
+        go-version: [1.20.x, 1.21.x]
```

Comment:
We are testing for 2 environments here, but only build for 1.21.x. Maybe a better idea to just test for 1.21.x, as that is also what we are distributing our releases with? I don;t think older versions of Go matter very much in that context